src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -45,6 +45,7 @@
                         </controls:SettingExpander.Header>
                         <controls:SettingExpander.Content>
                             <StackPanel>
+                                <CheckBox x:Uid="PowerLauncher_UseCentralizedKeyboardHook" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.UseCentralizedKeyboardHook}" Margin="{StaticResource ExpanderSettingMargin}" />
                                 <CheckBox x:Uid="PowerLauncher_IgnoreHotkeysInFullScreen" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.IgnoreHotkeysInFullScreen}" Margin="{StaticResource ExpanderSettingMargin}" />
```

Comment:
There's a separator line missing in between the CheckBoxes (line 49):

    <Rectangle Style="{StaticResource ExpanderSeparatorStyle}" />

Reply:
Thank you.
Added a commit for this.

Reply:
![image](https://user-images.githubusercontent.com/26118718/135627045-75ad9884-7c1d-4aa0-b0ce-4b3cd9fff8ff.png)